.github/workflows/mavenLicenseCheck.yml

```
@@ -50,29 +50,43 @@ on:
         required: false
 
 jobs:
-  check-licenses:
-    if: github.event_name != 'issue_comment' || ( github.event.issue.pull_request != '' && (github.event.comment.body == '/request-license-review') )
+  check-request:
     # Run on all non-comment events specified by the calling workflow and for comments on PRs that have a corresponding body.
+    if: >
+      github.event_name != 'issue_comment' ||
+      (github.event.issue.pull_request &&
+         (contains(github.event.comment.body, '/request-license-review') ||
+          contains(github.event.comment.body, '/license-check')))
```

This should stay strict in order to avoid accidental triggers.
Otherwise for example I cannot even simply explain in a comment how to request a review without triggering a check.
```suggestion
         (github.event.comment.body == '/request-license-review') ||
          (github.event.comment.body == '/license-check')))
```

---

makes sense, I usually do a full match, in this case I copied the conditions from the eclipse-set project.

---

Understand.
Btw. looks like I forgot an opening brace at the first operand of the `or` in my suggestion.
This has to be added to not break it.